worldedit-core/src/main/java/com/sk89q/worldedit/world/snapshot/Snapshot.java

```diff
@@ -101,9 +103,9 @@ private ChunkStore internalGetChunkStore() throws IOException, DataException {
 
                 return chunkStore;
             }
-        } else if (file.getName().toLowerCase().endsWith(".tar.bz2")
-                || file.getName().toLowerCase().endsWith(".tar.gz")
-                || file.getName().toLowerCase().endsWith(".tar")) {
+        } else if (lowerCaseFileName.endsWith(".tar.bz2")
```

Comment:
Will this break if they legitimately have other locale characters in their path?